frontend/src/components/GraphPlayground.jsx

```diff
@@ -167,4 +118,23 @@ export const DisplayGraph/*: React.FC*/ = () => {
       <RandomCircleGraph />
     </SigmaContainer>
   );
-};
\ No newline at end of file
+};
+function sugiyamaLayout(edges) {
```

Comment:
## `sugiyamaLayout`
How are we using this?

I know that you and Tim were going to create your own library outside of `d3dag`, and I know you guys have been working on this for awhile, but what you have here seems ad hoc.